tools/py/Instanton_interpolation.py

```diff
@@ -103,40 +103,45 @@
         q = beads.q
         atom = beads._blist[0]
 
-    print ' '
-    print 'We have a half ring polymer made of %i beads and %i atoms.' % (nbeads, natoms)
-    print 'We will expand the ring polymer to get a half polymer of %i beads.' % nbeadsNew
+    print(' ')
+    print( 'We have a half ring polymer made of {} beads and {} atoms.'.format(nbeads, natoms))
+    print( 'We will expand the ring polymer to get a half polymer of {} beads.'.format( nbeadsNew))
 
-    # Compose the full ring polymer.
-    #q2 = np.concatenate((q, np.flipud(q)), axis=0)
+   
 
-    # Make the rpc step
-    #rpc = nm_rescale(2 * nbeads, 2 * nbeadsNew)
-    #new_q = rpc.b1tob2(q2)[0:nbeadsNew]
-    rpc = nm_rescale(nbeads, nbeadsNew, np.asarray(range(natoms)))  # We use open path RPC
+    ## Make the rpc step (standar)
+    # q2 = np.concatenate((q, np.flipud(q)), axis=0)   # Compose the full ring polymer.
+    # rpc = nm_rescale(2 * nbeads, 2 * nbeadsNew)
+    # new_q = rpc.b1tob2(q2)[0:nbeadsNew]
+    ## Make the rpc step (instanton)
+    # rpc = nm_rescale(nbeads, nbeadsNew,instanton=True)
+    # new_q = rpc.b1tob2(q )
+
+    ## Make the rpc step (open path)
+    rpc = nm_rescale(nbeads, nbeadsNew, np.asarray(range(natoms)))  
     new_q = rpc.b1tob2(q)
 
     # Print
-    out = open("NEW_INSTANTON.xyz", "w")
+    out = open("new_instanton.xyz", "w")
     for i in range(nbeadsNew):
         atom.q = new_q[i] / unit_to_internal("length", "angstrom", 1.0)  # Go back to angstrom
         print_file("xyz", atom, cell, out, title='cell{atomic_unit}  Traj: positions{angstrom}')
         # print_file("xyz",pos[0],cell,out,title='cell  }')
     out.close()
 
-    print 'The new Instanton geometry (half polymer) was generated'
-    print 'Check NEW_INSTANTON.xyz'
-    print ''
-    print 'Remeber to change the number of beads (%i) in your input' % nbeadsNew
-    print ''
+    print( 'The new Instanton geometry (half polymer) was generated')
+    print( 'Check new_instanton.xyz')
+    print( '')
+    print( 'Remeber to change the number of beads {} in your input'.format(nbeadsNew))
+    print( '')
```

Comment:
Can you explain this procedure in a couple more words for the user? Can we tell the user in which situations (in which further calculations) they will have to change the number of beads?

Reply:
Yes, what I meant is that you have the chance to make it easier to the user. For example: 'Remember to change the number of beads {} in your input when starting your new simulation with an increased number of beads'...

Reply:
I see. Now it reads.

print( "Don't forget to change the number of beads to the new value ({}) in your input file".format(nbeadsNew))
print( 'when starting your new simulation with an increased number of beads.' )